crates/solvers/src/boundary/baseline.rs

```diff
@@ -137,10 +137,12 @@ impl<'a> Solver<'a> {
     }
 }
 
-fn to_boundary_amms(liquidity: &[liquidity::Liquidity]) -> HashMap<TokenPair, Vec<Amm>> {
+fn to_boundary_amms(
```

Comment:
Now with most things renamed to onchain liquidity maybe `to_boundary_liquidity` is more appropriate.